NotebookExamples/csharp/Kernel features.ipynb

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This samples looks good to share.  Just need to make a tiny typo colelctions to collections.
![image](https://user-images.githubusercontent.com/2546640/64440364-d1cda600-d099-11e9-941f-bec24bc00793.png)